src/js/common/schemaform/helpers.js

```diff
@@ -254,8 +254,8 @@ export function stringifyFormReplacer(key, value) {
     }
 
     // Exclude file data
-    if (value.confirmationCode && value.serverPath) {
-      return _.omit(['serverPath', 'serverName'], value);
+    if (value.confirmationCode && (value.serverPath || value.file)) {
```

Comment:
What does `stringifyFormReplacer` do?

Reply:
This is used by `transformForSubmit` when stringifying the form data to be submitted to the backend. It's the second parameter for `JSON.stringify`, which lets you modify the data as it's being stringified.